application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/excel/SpreadSheetImportTest.java

```diff
@@ -282,11 +282,13 @@ private void assertFieldTypes() {
     }
 
     private void assertListFieldTypes(List<Map<String, Object>> fieldTypes) {
-
         assertThat(fieldTypes,
                    allOf(hasItem(allOf(hasColumn("jurisdiction_id", getIdForTestJurisdiction()),
                                        hasColumn("reference", "FixedList-marritalStatusEnum"),
                                        hasColumn("base_field_type_id", fieldTypesId.get("FixedList")))),
+                       hasItem(allOf(hasColumn("jurisdiction_id", getIdForTestJurisdiction()),
+                                       hasColumn("reference", "DynamicList-marritalStatusEnum"),
+                                       hasColumn("base_field_type_id", fieldTypesId.get("FixedList")))),
                          hasItem(allOf(hasColumn("jurisdiction_id", getIdForTestJurisdiction()),
                                        hasColumn("reference", "MultiSelectList-marritalStatusEnum"),
```

Comment:
furthermore, is it possible to add some more unit tests rather than relying mostly on this test here which is an end to end test